core/bubble_dragger.js

```diff
@@ -95,6 +111,8 @@ Blockly.BubbleDragger.prototype.startBubbleDrag = function() {
   if (this.dragSurface_) {
     this.moveToDragSurface_();
   }
+
+  this.draggingBubble_.setDragging && this.draggingBubble_.setDragging(true);
```

Comment:
This line isn't specific to deletion.  Was it just missing before?

Reply:
Yeah it's missing from the bubble class. Should I add it there instead?

Reply:
No, this is fine.  Just wanted to make sure I understood.